public_suffix_list.dat

```diff
@@ -12988,6 +12988,14 @@ remotewd.com
 // Submitted by Yuvi Panda <yuvipanda@wikimedia.org>
 wmflabs.org
 
+// WoltLab GmbH : https://www.woltlab.com
+// Submitted by Tim Düsterhus <security@woltlab.cloud>
+community-pro.de
```

Comment:
As per https://github.com/publicsuffix/list/wiki/Guidelines, can you please sort the entries from right-to left.

Reply:
Sure. I've amended my commit and force pushed. I'm sorry, apparently I did not read that sorting part properly.